### DISCUSSION TITLE
Undo for clear annotate (and/or auto screenshot it)

### DISCUSSION THREAD

**Discussion by romanbarczynski** [Softal-io/presentify-community#16] [Ideas]
Hi, today I forgot to take screenshot of note before clearing annotate. I'd love to be able undo that clear

**Comment by rampatra** [comment]
You can undo it currently with Cmd + Z and Cmd + Shift + Z. Have you tried it?

**Comment by romanbarczynski** [comment]
Aah, so for anyone else having same issue:

Clear works like undo all, then you can "re-enter" it via "Redo": Cmd + Shift + Z - which goes step by step. Good enough
